bokehjs/src/coffee/core/bokeh_events.ts

```diff
@@ -0,0 +1,200 @@
+import { logger } from "./logging"
+
+export class Event {
+    static _event_classes = {}
+
+    constructor(options = {}) {
+        this._options = options;
+        if (options.model_id) {
+            this.model_id = null
+        }
+    }
```

Comment:
Well spotted - I messed up the conversion here. The code was working because when the ``model_id`` is null, it is broadcast to all the models.